### PR TITLE
Fix share action sheet options converting

### PR DESCRIPTION
**Pull request by sck-v**
There was error converting options for share action sheet between Reason's variant type and object, containing `url` or `message` fields required for native function

### CODE REVIEW

File: src/actionSheetIOS.re [hunk 1]
```diff
@@ -1,6 +1,8 @@
 type actionSheetConfig;
 
 type shareActionSheetConfig;
+external shareActionSheetConfig: Js.t('a) => shareActionSheetConfig =
+  "%identity";
```
Comment (czystyl) [inline src/actionSheetIOS.re]: `%` is added intentionally?

Comment (MoOx) [reply src/actionSheetIOS.re]: Probably, see 
https://bucklescript.github.io/docs/en/intro-to-external#special-identity-external

**Comment by MoOx** [comment]
@sck-v can you confirm that the change is not a breaking change for the bs binding?

File: src/actionSheetIOS.re (hunk 1 above)
Comment (czystyl) [reply src/actionSheetIOS.re]: Nice, I didn't know about it. Thanks for the explanation ;)

**Comment by sck-v** [comment]
> @sck-v can you confirm that the change is not a breaking change for the bs binding?

Yep, it just fixes internal implementation

**Pull request merged**